alea/runner.py

```diff
@@ -164,33 +164,48 @@ def __init__(
             statistical_model_args.get("asymptotic_dof", 1),
         )
 
+    def pre_process_poi(self, value, name):
```

Comment:
`name` is a bit generic. Maybe something like `called_by_name` instead to make it a bit clearer? But I can also live with name :D

Reply:
I updated it to `attribute_name`.